Formula/kong.rb

```diff
@@ -3,26 +3,19 @@ class Kong < Formula
   homepage "https://docs.konghq.com"
 
   stable do
-    url "https://bintray.com/kong/kong-src/download_file?file_path=kong-2.1.0.tar.gz"
-    sha256 "8e7ceae26a2755225e085f4b95a6972778d339dd877d3d2165a672117e6caf69"
-    depends_on "kong/kong/openresty@1.15.8.3"
+    url "https://bintray.com/kong/kong-src/download_file?file_path=kong-2.1.1.tar.gz"
   end
 
-  #devel do
-  #  url "https://github.com/Kong/kong.git", :tag => "2.1.0-rc.1"
-  #end
-
   head do
     url "https://github.com/Kong/kong.git", :branch => "next"
-    depends_on "kong/kong/openresty@1.15.8.3"
   end
 
-  depends_on "libyaml"
```

Comment:
this is installed by openresty_build_tools

Reply:
Have we tested DB-less loading without the `libyaml`? I think this is needed by `lyaml`.

Reply:
it is needed by `libyaml`, the CI might not be testing DB-less.

Reply:
the travis-ci tests are declarative I believe https://github.com/Kong/homebrew-kong/blob/master/.travis.yml#L33

Reply:
According to my tests locally this is still needed for real OS X environment (as suspected). I think the Travis environment might come preinstalled with `libyaml`, not so much for ordinary Mac.

```
==> Installing kong/kong/kong
==> Patching
patching file kong/templates/kong_defaults.lua
==> /usr/local/opt/openresty@1.15.8.3/luarocks/bin/luarocks --tree=/usr/local/Cellar/kong/2.1.3 make CRYPTO_DIR=/usr/local/opt/openresty@1.15.8.3/openssl OPENSSL
Last 15 lines from /Users/datong/Library/Logs/Homebrew/kong/01.luarocks:
kong 2.1.3-0 depends on lyaml 6.2.5 (not installed)
Installing https://luarocks.org/lyaml-6.2.5-1.src.rock

Error: Failed installing dependency: https://luarocks.org/lyaml-6.2.5-1.src.rock - Could not find library file for YAML
  No file libyaml.a in /usr/local/lib
  No file libyaml.dylib in /usr/local/lib
  No file matching libyaml.so.* in /usr/local/lib
  No file libyaml.a in /usr/lib
  No file libyaml.dylib in /usr/lib
  No file matching libyaml.so.* in /usr/lib
  No file libyaml.a in /lib
  No file libyaml.dylib in /lib
  No file matching libyaml.so.* in /lib
You may have to install YAML in your system and/or pass YAML_DIR or YAML_LIBDIR to the luarocks command.
Example: luarocks install lyaml YAML_DIR=/usr/local

If reporting this issue please do so to (not Homebrew/brew or Homebrew/core):
  kong/kong
```